daffodil-core/src/test/resources/test/example-of-most-dfdl-constructs.dfdl.xml

```diff
@@ -249,7 +249,8 @@
     <annotation>
       <appinfo source="http://www.ogf.org/dfdl/">
         <dfdl:element nilKind="literalValue" />
-       <!--  Gives a not yet implemented, so commented out.
+       <!--  Gives a SDE: newVariableInstance may only be used on
+            group reference, sequence or choice, so needs review (5/18/20).
```

Comment:
This should probably be moved to a model group in this file. Doesn't neccesarily need to be done as part of this commit. Could be done as a new PR as an update to DAFFODIL-341 along with the other newVariableInstance changes.

Reply:
That is correct. The DFDL Spec specifies that newVariableInstance may only be used on group reference, sequence, or choice. The correct place for the newVariableInstance call would probably be within the "gr" group declaration on line 175